docs/clients/go-client-api.md

```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
 
 RawKVClient is a client of the TiKV server and only supports the GET/PUT/DELETE/SCAN commands. The RawKVClient can be safely and concurrently accessed by multiple goroutines, as long as it is not closed. Therefore, for one process, one client is enough generally.
 
+### Possiable Error
+
+If you get the error like follows.
```

Comment:
em, I think we should use update the doc to use tikv client directly

@disksing